datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -1327,46 +1327,173 @@ select avg(c1), arrow_typeof(avg(c1)) from d_table
 ----
 5 Decimal128(14, 7)
 
-# FIX: different test table
+
 # aggregate
-# query I
-# SELECT SUM(c1), SUM(c2) FROM test
-# ----
-# 60 220
+query II
+SELECT SUM(c1), SUM(c2) FROM test
+----
+7 6
```

Comment:
This gets a different answer (likely because the table is different). However, I think the test still offers the same coverage.

Reply:
Exactly, I believed that the if the old tests can be trusted, then this one having a different result is trustable